python/tskit/util.py

```diff
@@ -45,9 +45,18 @@ def canonical_json(obj):
 def is_unknown_time(time):
     """
     As the default unknown mutation time is NAN equality always fails. This
-    method compares the bitfield.
+    method compares the bitfield such that unknown times can be detected.
+    Either single floats can be passed or lists/arrays.
+
+    :param float or array-like time: Value or array to check.
+    :return: A single boolean or array of booleans the same shape as ``time``.
+    :rtype: bool or np.array(dtype=bool)
     """
-    return struct.pack(">d", UNKNOWN_TIME) == struct.pack(">d", time)
+    try:
+        return struct.pack(">d", UNKNOWN_TIME) == struct.pack(">d", time)
```

Comment:
Do we need to first part now? I thought numpy would cast to a 1-element array.

Reply:
Seems good - I get a `bool` back! Nice.